spdx-utils/src/main/kotlin/SpdxLicenseMapping.kt

```diff
@@ -19,337 +19,331 @@
-            "GNU General Public License v2 or later (GPLv2+)" to enumSetOf(SpdxLicense.GPL_2_0PLUS),
-            "GNU General Public License v3 (GPLv3)" to enumSetOf(SpdxLicense.GPL_3_0),
-            "GNU General Public License v3 or later (GPLv3+)" to enumSetOf(SpdxLicense.GPL_3_0PLUS),
+ """.trimIndent() to APACHE_2_0.toExpression(),
+            "Apache2" to APACHE_2_0.toExpression(),
+            "Artistic 2.0" to ARTISTIC_2_0.toExpression(),
+            "BOOST" to BSL_1_0.toExpression(),
+            "BSD" to BSD_2_CLAUSE.toExpression(),
+            "BSD (3-clause)" to BSD_3_CLAUSE.toExpression(),
+            "BSD - See ndg/httpsclient/LICENCE file for details" to BSD_3_CLAUSE.toExpression(),
+            "BSD 2-Clause" to BSD_2_CLAUSE.toExpression(),
+            "BSD 2-Clause License" to BSD_2_CLAUSE.toExpression(),
+            "BSD 3 Clause" to BSD_3_CLAUSE.toExpression(),
+            "BSD 3-Clause" to BSD_3_CLAUSE.toExpression(),
+            "BSD 3-Clause \"New\" or \"Revised\" License (BSD-3-Clause)" to BSD_3_CLAUSE.toExpression(),
+            "BSD 3-Clause License" to BSD_3_CLAUSE.toExpression(),
+            "BSD 3-Clause license" to BSD_3_CLAUSE.toExpression(),
+            "BSD 3-clause" to BSD_3_CLAUSE.toExpression(),
+            "BSD Licence 3" to BSD_3_CLAUSE.toExpression(),
+            "BSD License" to BSD_2_CLAUSE.toExpression(),
+            "BSD License for HSQL" to BSD_3_CLAUSE.toExpression(),
+            "BSD New" to BSD_3_CLAUSE.toExpression(),
+            "BSD New license" to BSD_3_CLAUSE.toExpression(),
+            "BSD licence" to BSD_2_CLAUSE.toExpression(),
+            "BSD license" to BSD_2_CLAUSE.toExpression(),
+            "BSD or Apache License, Version 2.0" to (BSD_2_CLAUSE or APACHE_2_0),
+            "BSD style" to BSD_3_CLAUSE.toExpression(),
+            "BSD style license" to BSD_3_CLAUSE.toExpression(),
+            "BSD*" to BSD_3_CLAUSE.toExpression(),
+            "BSD-3" to BSD_3_CLAUSE.toExpression(),
+            "BSD-Style" to BSD_3_CLAUSE.toExpression(),
+            "BSD-Style License" to BSD_3_CLAUSE.toExpression(),
+            "BSD-like" to BSD_3_CLAUSE.toExpression(),
+            "BSD-style" to BSD_3_CLAUSE.toExpression(),
+            "BSD-style license" to BSD_3_CLAUSE.toExpression(),
+            "BSD2" to BSD_2_CLAUSE.toExpression(),
+            "BSD3" to BSD_3_CLAUSE.toExpression(),
+            "Berkeley Software Distribution (BSD) License" to BSD_2_CLAUSE.toExpression(),
+            "Bouncy Castle Licence" to MIT.toExpression(),
+            "Bouncy Castle License" to MIT.toExpression(),
+            "CC0" to CC0_1_0.toExpression(),
+            "CC0 1.0 Universal" to CC0_1_0.toExpression(),
+            "CC0 1.0 Universal License" to CC0_1_0.toExpression(),
+            "CDDL" to CDDL_1_0.toExpression(),
+            "CDDL + GPLv2 with classpath exception" to (CDDL_1_0 and GPL_2_0_WITH_CLASSPATH_EXCEPTION),
```

Comment:
Shouldn't this be "GPL_2_0_ONLY with CLASSPATH_EXCEPTION_2_0" or so? I.e. shouldn't we prefer to map to non-deprecated SPDX expressions? This would require to also overload the `with` operator. Similar in more places below.

Reply:
Done.